core/components/crosscontextssettings/processors/mgr/settings/getvaluelist.class.php

```diff
@@ -58,8 +58,12 @@ public function getData() {
         /* query for chunks */
         $c = $this->modx->newQuery($this->classKey);
         $c = $this->prepareQueryBeforeCount($c);
-//        $data['total'] = $this->modx->getCount($this->classKey, $c);
-        $data['total'] = $this->modx->crosscontextssettings->getQueryCount($this->classKey, $c);
+        $version = $this->modx->getVersionData();
+        if ($version['major_version'] <= 2) {
+            $data['total'] = $this->modx->crosscontextssettings->getQueryCount($this->classKey, $c);
+        } else {
+            $data['total'] = $this->modx->getCount($this->classKey, $c);
```

Comment:
Not sure if getCount/groupBy is working in 2.3.x

Had some issues with getQueryCount in 2.3. But I don't know which.